cinq_auditor_domain_hijacking/__init__.py

```diff
@@ -67,18 +67,19 @@ def run(self, *args, **kwargs):
 
             for dist in dists:
                 for org in dist.origins:
-                    bucket = org.replace('.s3.amazonaws.com', '')
-
-                    if bucket not in buckets:
-                        key = '{} ({})'.format(bucket, dist.type)
-                        issues.append({
-                            'key': key,
-                            'value': 'S3Bucket {} doesnt exist on any known account. Referenced by {} on {}'.format(
-                                bucket,
-                                dist.domain_name,
-                                dist.account,
-                            )
-                        })
+                    if org['type'] == 's3':
+                        bucket = org['source'].replace('.s3.amazonaws.com', '')
+
+                        if bucket not in buckets:
+                            key = '{} ({})'.format(bucket, dist.type)
+                            issues.append({
+                                'key': key,
+                                'value': 'S3Bucket {} doesnt exist on any known account. Referenced by {} on {}'.format(
```

Comment:
IMO `S3 Bucket {} does not` is better

Reply:
We can change it, but then we should change it throughout the entire system, but I would prefer that to be done separate to this PR, so we keep the functional and cosmetic changes in separate PRs